src/SourceBuild/tarball/patches/runtime/0003-Enable-source-link-in-source-build-for-.version-file.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Davis Goodin <dagood@microsoft.com>
```

Comment:
@dsplaisted do we want to remove this, there's PII in this . Should credscan pick this up?

Reply:
I think this is how the source-build patches always look, and also that it's the same information that is normally in the git database, just in patch files in this case.

Reply:
ok, seems wrong though since it's publicly disclosing email addresses